<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reduction_options</class>
 <widget class="QDialog" name="reduction_options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>reduction options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>low wavelength</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="low_wavelength">
     <property name="specialValueText">
      <string/>
     </property>
     <property name="maximum">
      <double>25.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>2.500000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>high wavelength</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="high_wavelength">
     <property name="specialValueText">
      <string/>
     </property>
     <property name="minimum">
      <double>2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>25.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>19.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>rebin percentage</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="rebin_percent">
     <property name="specialValueText">
      <string/>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>expected centre</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="expected_centre">
     <property name="specialValueText">
      <string/>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>1024.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>500.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="background_subtraction">
     <property name="text">
      <string>background subtraction</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="manual_beam_find">
     <property name="text">
      <string>manual beam find</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="monitor_normalisation">
     <property name="text">
      <string>normalise by beam monitor</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="save_spectrum">
     <property name="text">
      <string>save spectrum</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="save_offspecular">
     <property name="text">
      <string>save offspecular map</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="streamed_reduction">
     <property name="text">
      <string>streamed reduction</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>reduction_options</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>reduction_options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
